docs/index.asciidoc

```diff
@@ -33,6 +33,23 @@ Example use-cases:
 
 Note: While this project is partially maintained by Google, this is not an official Google product.
 
+.Installation Note
+[NOTE]
+================================================================================
+When installing this plugin you could receive an error message about the
+impossibility to find compatible versions for gem "mimemagic".
+This is error could be circumvented by installing the required dependency directly into
+the Logstash Ruby gem's cache.
+
+This could be done going into Logstash installation folder and using the bundled Ruby gems instance:
+
+[source,bash]
+----------------------------------
+bin/ruby -S gem install mimemagic -v '>= 0.3.7'
+----------------------------------
+
+================================================================================
+
```

Comment:
Hi @karenzone , I've added this note to describe how to avoid an installation problem. Would be great if you could review it, or ping me if you need more context on the problem we are facing.

Reply:
@andsel I would do something like this:

Attempting to install this plugin may result in an error about "<post here a sample error that you actually get from installation".

If this error occurs, you can fix it by manually installing the "mime magic" dependency directly into the Logstash Ruby Gem's cache (<provide a link here to where that would be, or the location.). This could be done using the bundled Ruby gems instance inside the Logstash installation folder (< provide references, e.g. a link that shows where things are installed based on the installation  method>)

Reply:
That's a good idea, I rephrased it in 64647d0949537a34fa078cd76c53eb25e9f2f61d